src/KafkaFlow.Abstractions/AutoOffsetReset.cs

```diff
@@ -1,8 +1,12 @@
 namespace KafkaFlow
 {
+    /// <summary>AutoOffsetReset enum values</summary>
     public enum AutoOffsetReset
     {
+        /// <summary>Latest</summary>
         Latest,
+        
+        /// <summary>Earliest</summary>
```

Comment:
In WithAutoOffsetReset method there is a description of each one, you can remove from there and put it here